internal/use_cases/create_pull_request.go

```diff
@@ -43,8 +48,50 @@ type CreatePullRequest struct {
 	BranchProvider          domain.BranchProvider
 }
 
+func normalizeTemplatePath(git domain.GitProvider, path string) (string, error) {
+	if path == "" {
+		return "", nil
+	}
+
+	if !filepath.IsAbs(path) {
+		repoRoot, err := git.GetRepositoryRoot()
```

Comment:
suggestion: use provider within the struct once this function is a struct method.
```suggestion
		repoRoot, err := cpr.Git.GetRepositoryRoot()
```